src/components/Home/contact.css

```diff
@@ -16,15 +15,30 @@
   padding-top: 17px;
 }
 
-.send-button{
-  margin-top: 19px;
-  margin-bottom: 20px;
+.send-card{
+  height: 40px;
+  border-radius: 4px;
+  background-color: #f9f8f8;
+  border: solid 1px #ef005c;
+  margin-top: 50px;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.send-card-text {
+  font-family: HelveticaNeue;
```

Comment:
How come we have to do this? aren't we using font fam with fall backs for entire app?